cloudca/resource_cloudca_port_forwarding_rule.go

```diff
@@ -45,26 +44,26 @@ func resourceCloudcaPortForwardingRule() *schema.Resource {
 				Description: "The protocol that this rule should use (eg. TCP, UDP)",
 			},
 			"private_port_start": {
-				Type:        schema.TypeInt,
+				Type:        schema.TypeString,
```

Comment:
Have you tried to pass an integer into this from terraform side? I mean do we know if this is backward compatible?

Reply:
Yes I have tested and validated both scenarios. Notice the details in the PR description.

Reply:
@simongodard thanks for pointing that out. I will have a look and see if I can include fixes for those at the same time.

Reply:
haha, I should have read the description more closely! :+1: